### ISSUE TITLE
`npm i` is failing with node8 + npm5

### ISSUE THREAD

**Issue by dopeddude** [strongloop/loopback#3427]
# Description/Steps to reproduce

```
$ npm i loopback
npm ERR! finishInflating is not a function

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/******/.npm/_logs/2017-06-02T07_04_51_942Z-debug.log

$ tail -n 35 /Users/******/.npm/_logs/2017-06-02T07_03_56_716Z-debug.log
629 silly pacote range manifest for assert-plus@^1.0.0 fetched in 2ms
630 silly resolveWithNewModule assert-plus@1.0.0 checking installable status
631 http fetch GET 200 https://registry.npmjs.org/punycode 2ms (from cache)
632 silly pacote range manifest for punycode@^1.4.1 fetched in 3ms
633 silly resolveWithNewModule punycode@1.4.1 checking installable status
634 http fetch GET 200 https://registry.npmjs.org/options 2ms (from cache)
635 silly pacote tag manifest for options@latest fetched in 3ms
636 silly resolveWithNewModule options@0.0.6 checking installable status
637 silly pacote range manifest for js2xmlparser@^2.0.2 fetched in 1ms
638 silly resolveWithNewModule js2xmlparser@2.0.2 checking installable status
639 verbose stack TypeError: finishInflating is not a function
639 verbose stack     at module.exports (/usr/local/lib/node_modules/npm/lib/install/inflate-shrinkwrap.js:23:12)
639 verbose stack     at /usr/local/lib/node_modules/npm/lib/install/deps.js:615:16
639 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/iferr/index.js:13:50
639 verbose stack     at addBundled (/usr/local/lib/node_modules/npm/lib/fetch-package-metadata.js:93:78)
639 verbose stack     at /usr/local/lib/node_modules/npm/lib/install/deps.js:587:5
639 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/iferr/index.js:13:50
639 verbose stack     at checkPlatform (/usr/local/lib/node_modules/npm/node_modules/npm-install-checks/index.js:53:10)
639 verbose stack     at thenWarnEngineIssues (/usr/local/lib/node_modules/npm/lib/install/validate-args.js:41:5)
639 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/iferr/index.js:13:50
639 verbose stack     at checkEngine (/usr/local/lib/node_modules/npm/node_modules/npm-install-checks/index.js:10:20)
639 verbose stack     at module.exports.isInstallable (/usr/local/lib/node_modules/npm/lib/install/validate-args.js:38:3)
639 verbose stack     at resolveWithNewModule (/usr/local/lib/node_modules/npm/lib/install/deps.js:586:10)
639 verbose stack     at /usr/local/lib/node_modules/npm/lib/install/deps.js:532:7
639 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/iferr/index.js:13:50
639 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/call-limit/call-limit.js:22:12
639 verbose stack     at _combinedTickCallback (internal/process/next_tick.js:95:7)
639 verbose stack     at process._tickCallback (internal/process/next_tick.js:161:9)
640 verbose cwd /Users/ashwinikumar/hotelsoft/hotelsoftrzr/hs-srvcommon
641 verbose Darwin 16.1.0
642 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "i" "loopback"
643 verbose node v8.0.0
644 verbose npm  v5.0.0
645 error finishInflating is not a function
646 verbose exit [ 1, true ]
```

# Expected result

The successful installation

# Additional information

This seems to be failing with node8.0.0 + npm5.0.0

**Comment by dopeddude** [comment]
Seems like the issue lies somewhere inside npm. I have submitted a PR here https://github.com/npm/npm/pull/16908

But, I am still curious to know, what is different with this module `strongloop/strong-remoting` which is not letting the `npm i loopback` be successful. As other modules are getting installed successfully.

**Comment by shaheero** [comment]
Maybe it's better to create the issue here?
https://github.com/strongloop/strong-remoting/issues

**Comment by Setogit** [comment]
Successfully installed `loopback@3.8.0`in my OSX environment.  I'm using noe 8.0.0 and npm 5.0.2.

@dopeddude -- do you see the same issue with npm 5.0.2?

$ npm i loopback
npm WARN tmp-main@9.9.9 No description
npm WARN tmp-main@9.9.9 No repository field.
npm WARN tmp-main@9.9.9 No license field.

+ loopback@3.8.0
added 240 packages in 11.314s
$ node -v
v8.0.0
$ npm -v
5.0.2

**Comment by dopeddude** [comment]
The issue seems fixed with `npm 5.0.2`. Thanks.
